packager/hls/public/hls_params.h

```diff
@@ -56,6 +56,8 @@ struct HlsParams {
   /// be populated from segment duration specified in ChunkingParams if not
   /// specified.
   double target_segment_duration = 0;
+  /// This allows to change the initial EXT-X-MEDIA-SEQUENCE field value.
```

Comment:
Change to something like: (Also note 80 line length limit)

/// Custom EXT-X-MEDIA-SEQUENCE value to allow continuous media playback across packager restarts. See https://github.com/google/shaka-packager/issues/691 for details.